### PR TITLE
docs: improve component event doc

### PR DESCRIPTION
**Pull request by hamatoyogi**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

### CODE REVIEW

File: packages/docs/src/routes/docs/components/events/index.mdx [hunk 1]
```diff
@@ -188,11 +211,18 @@ Notice that `on:click` attribute contains three pieces of information:
 2. `Counter_button_onClick`: The symbol which needs to be retrieved from the lazy-loaded chunk.
 3. `[0]`: An array of lexically captured variable references (State of the closure).
 
-In our case `() => store.count++` only captures `store`, and hence it contains only a single reference `0`. `0` is an index into the `q:obj` attribute that contains a reference to the actual serialized object, referring to `store`. (The exact mechanisms and syntax is an implementation detail that can change at any time)
+In our case `() => store.count++` only captures `store`, and hence it contains only a single reference `0`. 
+
+`0` is an index into the `q:obj` attribute that contains a reference to the actual serialized object, referring to `store` (The exact mechanisms and syntax is an implementation detail that can change at any time).
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/components/events/index.mdx:216]: this is actually super old, outdated and too implementation detail, we should remove this, but i am ok to leave it for another PR

**Comment by manucorporat** [comment]
Ship it!

File: packages/docs/src/routes/docs/components/events/index.mdx (hunk 1 above)
Comment (hamatoyogi) [reply packages/docs/src/routes/docs/components/events/index.mdx:216]: Fine with removing it @manucorporat, but which part?
